javascript/apps/taiga/src/app/shared/comments/components/comment-detail/comment-detail.component.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023-present Kaleidos INC
+ */
+
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  DestroyRef,
+  EventEmitter,
+  Input,
+  Output,
+  inject,
+} from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { TRANSLOCO_SCOPE, TranslocoModule } from '@ngneat/transloco';
+import { Store } from '@ngrx/store';
+import { RxState } from '@rx-angular/state';
+import { TuiAutoFocusModule } from '@taiga-ui/cdk';
+import {
+  TuiButtonModule,
+  TuiDataListModule,
+  TuiSvgModule,
+} from '@taiga-ui/core';
+import { ShortcutsService } from '@taiga/core';
+import { Project, User, UserComment } from '@taiga/data';
+import { Subject, takeUntil } from 'rxjs';
+import { selectCurrentProject } from '~/app/modules/project/data-access/+state/selectors/project.selectors';
+import { DropdownModule } from '~/app/shared/dropdown/dropdown.module';
+import { filterNil } from '~/app/shared/utils/operators';
+
+@Component({
+  selector: 'tg-comment-detail',
+  standalone: true,
+  templateUrl: './comment-detail.component.html',
+  styleUrls: ['./comment-detail.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [
+    CommonModule,
+    DropdownModule,
+    TranslocoModule,
+    TuiSvgModule,
+    TuiDataListModule,
+    TuiButtonModule,
+    TuiAutoFocusModule,
+  ],
+  providers: [
+    RxState,
+    {
+      provide: TRANSLOCO_SCOPE,
+      useValue: 'comments',
+    },
+  ],
+})
+export class CommentDetailComponent {
+  @Input({ required: true }) public comment!: UserComment;
+  @Input({ required: true }) public canComment!: boolean;
+  @Input({ required: true }) public user!: User;
+
+  @Output() public highlightComment = new EventEmitter<string | undefined>();
+  @Output() public deleteComment = new EventEmitter<string>();
+
+  public store = inject(Store);
+  public state = inject(
+    RxState<{
+      project: Project;
+    }>
+  );
+  public shortcutsService = inject(ShortcutsService);
+  public destroyRef = inject(DestroyRef);
+
+  public model$ = this.state.select();
+
+  public showDeleteCommentConfirm = false;
+  public commentOptionsState = false;
+
+  private unsetDelete$: Subject<void> = new Subject();
+
+  constructor() {
+    this.state.connect(
```

Comment:
You can do `public project = this.store.selectSignal(selectCurrentProject);` so you won't need the rxstate
At the template just use it like `project().userIsAdmin`

Reply:
you forgot to delete the rxstate, it's not used in the file anymore